extensions/mssql/src/utils.ts

```diff
@@ -31,6 +31,16 @@ export function getAppDataPath() {
 	}
 }
 
+/**
+ * This function returns whether the native credential management system
+ * should be used instead of tools service
+ */
+export function nativeCredentialsEnabled() {
+	const linux: boolean = os.platform() === 'linux';
+	const nativeCredentialsEnabled: boolean = true; // add setting for this after review
```

Comment:
we should make this a user setting.  also we should default to false until we have the ability to move deal with existing credentials that need to be migrated.  if we aren't going to support credential migration, we should prompt the user if they have existing passwords and ask if the want to switch to this new mechanism.